pkg/config/config.go

```diff
@@ -720,6 +720,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("compliance_config.enabled", true)
 	config.BindEnvAndSetDefault("compliance_config.check_interval", 20*time.Minute)
 	config.BindEnvAndSetDefault("compliance_config.dir", "/etc/datadog-agent/compliance.d")
+	config.BindEnvAndSetDefault("compliance_config.cmd_port", 5010)
```

Comment:
let's make sure `5010` isn't used by any popular software to avoid potential collision especially when the agent is running in non containerized environments

Reply:
Thank you, good point, so I am seeing it here - https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=1&page=42, this sounds pretty obscure in my opinion.

Currently these are used for node and DCA:
```
	config.BindEnvAndSetDefault("cmd_port", 5001)
	config.BindEnvAndSetDefault("cluster_agent.cmd_port", 5005)
```
We will provide the way to override the default port.